alfresco-transformer-base/src/main/java/org/alfresco/transformer/clients/AlfrescoSharedFileStoreClient.java

```diff
@@ -2,7 +2,7 @@
  * #%L
```

Comment:
1. I don't think there should be any changes in this file as it is to do with the SFS and your change is to do with DAUs which is an alternative. Yes there may be commonality between the two classes, but it would be better to have a separate class. It might (only might) make sense to have a common abstract parent or share a component.